Added an option to start maximized

acidrain42 opened the pull request.

danbruegge commented:
Why isn't that merged? I think it is such a useful feature.

thestinger commented:
Is there a use case for the configuration option? I can understand wanting the command-line switch, but not the other part.

acidrain42 commented:
I added the option because I want termite to always start maximized. In my case, I don't need the command-line switch but I added it for consistency and because I figured other people might find it useful.

jelly commented:
Not sure if @thestinger  wants this still merged, but you are missing atleast the man page changes.

acidrain42 commented:
@jelly Good point, I updated the man pages just in case.

Marcel-Robitaille commented:
Has there been any progress on this? I love termite and use it every day. I would really like this feature.

polyzen commented on config:
```diff
@@ -5,6 +5,7 @@
 #dynamic_title = true
 font = Monospace 9
 #fullscreen = true
+maximized = false
```
Please comment this out as it's the default anyhow.

acidrain42 replied:
Done

Iambecomeroot commented:
Any idea when well see this get merged?

jelly commented:
I agree with @thestinger I think it's sensible as a command line option, since that can be easily bound to a keybinding. Otherwise re-open it and we can continue the diiscussion.

Iambecomeroot commented:
Seems reasonable. I use a tiling wm now so I don't care anymore.

b-fg commented:
I would like this feature as well, even if only as a command line option.
